src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -99,6 +100,12 @@ namespace OpenMS
     */
     void searchModifications(std::set<const ResidueModification*>& mods, const String& mod_name, const String& residue = "", ResidueModification::TermSpecificity term_spec = ResidueModification::NUMBER_OF_TERM_SPECIFICITY) const;
 
+    const ResidueModification* searchModifications_fast(const String& mod_name, 
```

Comment:
more style conform would be searchModificationsFast
and some docs would need to be added